### PR TITLE
re org of the readme

### PR DESCRIPTION
**Pull request by joshuafernandes**
- Simplified it a good bit, and put a TOC at the top
- Each example has an individual 'start by' 
- Also re did the use cases for each so its easier to follow

View the new layout here https://github.com/joshuafernandes/besu-sample-networks/tree/toc 

Welcome any thoughts / feedback on this

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,31 +29,67 @@ To run these tutorials, you must have the following installed:
...
 
 
-## Description
+## Example Network Setups
+All our documentation can be found on the [Besu documentation site](https://besu.hyperledger.org/Tutorials/Examples/Private-Network-Example/).
 
-There are multiple examples in this repo, and each has a POW and POA example. You can choose the default setup which comprises a 4 node network with the Block Explorer and our Prometheus & Grafana dashboard to track the progress of the chain; or the whole setup with logging via ELK.   
-Please use the following use cases personas as guidelines only: 
+There are multiple examples in this repo, and each has a POW(proof of work) and POA(proof of authority) setup. Each setup 
+comprises a minimum of 4 ethereum nodes with monitoring tools like:
```
Comment (bgravenorst) [inline README.md:36]:
```suggestion
There are multiple examples in this repo, and each has a Proof Of Work (POW) and Proof of Authority (POA) setup. Each setup 
comprises a minimum of 4 Ethereum nodes with monitoring tools like:
```

File: README.md [hunk 2]
```diff
@@ -18,31 +29,67 @@ To run these tutorials, you must have the following installed:
...
+- [Metrics monitoring](https://besu.hyperledger.org/en/stable/HowTo/Monitor/Metrics/) via prometheus and grafana to give you insights into how the chain is progressing
+- Optional [logs monitoring](https://besu.hyperledger.org/en/latest/HowTo/Monitor/Elastic-Stack/) to give you real time logs of the nodes. This feature is enabled with a `-e` flag when starting the sample network
 
-All our documentation can be found on the [Besu documentation site](https://besu.hyperledger.org/Tutorials/Examples/Private-Network-Example/).
+The examples also include architecture diagrams to visually show components. They generally use the POA (IBFT2 algorithm) setup, and to view the architecture diagrams for the POW (ethash) setup please see the `images` folder (where the POA variants have different suffixes). 
 
-All the architecture diagrams below generally use the POA (IBFT2) setup, to view the architecture diagrams for the POW (ethash) setup please see the `images` folder, where the files share the same name but have different suffixes. 
+Each section also includes a few use cases personas (intended as guidelines only).
```
Comment (bgravenorst) [inline README.md:43]:
```suggestion
Each section also includes use case personas (intended as guidelines only).
```

File: README.md [hunk 3]
```diff
@@ -18,31 +29,67 @@ To run these tutorials, you must have the following installed:
...
  
-![Image basic](./images/sampleNetworks-poa.png)
+![Image basic_pow](./images/sampleNetworks-pow.png)
+
+Start the network with: 
+
+`./run.sh ` 
+
+Use cases: 
+ - you are learning about how ethereum works
```
Comment (bgravenorst) [inline README.md:72]:
```suggestion
 - you are learning about how Ethereum works 
```

File: README.md [hunk 4]
```diff
@@ -18,31 +29,67 @@ To run these tutorials, you must have the following installed:
...
+### ii. POA (ethash) Network  <a name="poa-network"></a>
+ 
+![Image basic_poa](./images/sampleNetworks-poa.png)
+
+Start the network with: 
+
+`./run.sh -c ibft2` 
+
+Use cases: 
+ - you are learning about how ethereum works
```
Comment (bgravenorst) [inline README.md:86]:
```suggestion
 - you are learning about how Ethereum works 
```

File: README.md [hunk 5]
```diff
@@ -18,31 +29,67 @@ To run these tutorials, you must have the following installed:
...
+ 
+![Image basic_poa](./images/sampleNetworks-poa.png)
+
+Start the network with: 
+
+`./run.sh -c ibft2` 
+
+Use cases: 
+ - you are learning about how ethereum works 
+ - you are looking to create a private eth network
```
Comment (bgravenorst) [inline README.md:87]:
```suggestion
 - you are looking to create a private Ethereum network
```

File: README.md [hunk 6]
```diff
@@ -54,28 +101,39 @@ Behind the scenes, this has used a smart contract that is compiled and then depl
...
+Start the network with: 
 
-#### Basic 4 Node Network with Privacy (Orion) node sets, the Block Explorer and Prometheus & Grafana 
-Use this scenario:
-- if you are a user looking to execute private transactions at least one other party
+`./run.sh -c ibft2 -e` 
+
+Use cases: 
+ - you are learning about how ethereum works 
+ - you are looking to create a private eth network
```
Comment (bgravenorst) [inline README.md:116]:
```suggestion
 - you are learning about how Ethereum works 
 - you are looking to create a private Ethereum network
```

File: README.md [hunk 7]
```diff
@@ -54,28 +101,39 @@ Behind the scenes, this has used a smart contract that is compiled and then depl
...
 
-![Image basic_orion](./images/sampleNetworks-poa-orion.png)
- 
 `./run-privacy.sh` starts all the docker containers in POW mode, and also has 3 Orion nodes for privacy 
- 
-`./run-privacy.sh -c ibft2` starts all the docker containers in POA mode using the IBFT2 Consensus algorithm, and also has 3 Orion nodes for privacy 
 
-There is an optional `-e` parameter which provides centralised logging functionality via ELK 
+Use cases: 
+ - you are learning about how eth works
```
Comment (bgravenorst) [inline README.md:130]:
```suggestion
 - you are learning about how Ethereum works 
```

File: README.md [hunk 8]
```diff
@@ -54,28 +101,39 @@ Behind the scenes, this has used a smart contract that is compiled and then depl
...
- 
 `./run-privacy.sh` starts all the docker containers in POW mode, and also has 3 Orion nodes for privacy 
- 
-`./run-privacy.sh -c ibft2` starts all the docker containers in POA mode using the IBFT2 Consensus algorithm, and also has 3 Orion nodes for privacy 
 
-There is an optional `-e` parameter which provides centralised logging functionality via ELK 
+Use cases: 
+ - you are learning about how eth works 
+ - you are a user looking to execute private transactions at least one other party
+ - you are looking to create a private eth network with private transactions between two or more parties. The logs make it easy to see whats going on betwen nodes and transactions
```
Comment (bgravenorst) [inline README.md:132]:
```suggestion
 - you are looking to create a private Ethereum network with private transactions between two or more parties. The logs make it easy to see whats going on between nodes and transactions
```

File: README.md [hunk 9]
```diff
@@ -101,23 +156,30 @@ There is an additional erc20 token example that you can also test with: executin
...
+`./run-permissioning.sh -e` gets the latest smart contract code, compiles the contracts and updates the genesis file with the contract code. Once done it spins up a full network 
+
+`./run-permissioning-dapp.sh -e` With the network up from the previous step, it will migrate the contracts to the network. Once complete, it restarts the blockchain network with permissions enabled so the rules and permissions deployed in the previous step take effect
+
+Open a new tab in your browser and go to `http://localhost:3001` to use the Permissioning DApp 
+
+
 Use this scenario:
-- if you are a DevOps engineer or administrator looking to see how the full blockchain works with on chain permissioning and retrictions
+ - if you are a DevOps engineer or administrator looking to see how the full blockchain works with on chain permissioning and retrictions
```
Comment (bgravenorst) [inline README.md:177]:
```suggestion
 - if you are a DevOps engineer or administrator looking to see how the full blockchain works with onchain permissioning and restrictions
```

File: README.md [hunk 10]
```diff
@@ -127,22 +189,21 @@ Use this scenario:
...
 
 
+Start the network with: 
 
+`./run.sh -c ibft2 -s` gets the latest smart contract code, compiles the contracts and updates the genesis file with the contract code. Once done it spins up a full network 
 
-### Stop Services and Network
-`./stop.sh` stops all the docker containers created.
+Use this scenario:
+ - if you need to sign transactions with a private key and forward that along to the eth client ie. Besu
```
Comment (bgravenorst) [inline README.md:206]:
```suggestion
 - if you need to sign transactions with a private key and forward that to the Ethereum client (for example Besu)
```

File: README.md (hunk 3 above)
Comment (joshuafernandes) [reply README.md:72]: :+1:

**Comment by bgravenorst** [comment]
Just minor nitpicks :)

File: README.md (hunk 1 above)
Comment (joshuafernandes) [reply README.md:36]: :+1:

File: README.md (hunk 2 above)
Comment (joshuafernandes) [reply README.md:43]: :+1:

**Comment by joshuafernandes** [comment]
> Just minor nitpicks :)

Thanks mate, all fixed up now :)

**Pull request merged**